core/src/main/java/c/org/rajawali3d/object/renderers/ObjectRendererImpl.java

```diff
@@ -192,28 +197,44 @@ private void checkDoubleSided(@NonNull ObjectRenderer lastUsed) {
     }
 
     private void checkBlending(@NonNull ObjectRenderer lastUsed) {
-        if (isBlended && !lastUsed.isBlended()) {
-            // We are blended, the last renderer is not - enable
-            GLES20.glEnable(GLES20.GL_BLEND);
-            GLES20.glBlendFunc(blendSourceFactor, blendDestinationFactor);
-        } else if (lastUsed.isBlended()) {
-            // We are not blended, the last renderer is - disable
-            GLES20.glDisable(GLES20.GL_BLEND);
+        if (isBlended) {
```

Comment:
Why was this expanded into more verbose logic?

Reply:
If  isBlended is true and lastUsed was also, then the else clause will be used.

Reply:
Same as above

Reply:
Same for me too